.eslintrc.js

```diff
@@ -20,11 +20,10 @@ module.exports = {
     'plugin:import/errors',
     'plugin:import/warnings',
     'plugin:import/typescript',
-    'plugin:jest/recommended',
     'prettier',
     'prettier/@typescript-eslint',
   ],
-  plugins: ['jest', '@typescript-eslint', 'import'],
+  plugins: ['@typescript-eslint', 'import'],
```

Comment:
Are there equivalent rules in `eslint-plugin-vitest`? I have not checked in depth, but might be worth looking at later.

Reply:
There are equivalents for most — I added it in my latest commits. The only ones that I didn't find equivalents for are: `prefer-todo` (but there are no usages of `test.todo`) and `require-to-throw-message` (disabled anyway).

Reply:
Yeah, sorry, pushed now.